reference/shipping.v3.yml

```diff
@@ -142,6 +143,61 @@ paths:
                     CA: '508313'
                     AU: '817355'
                     ALL: '501000'
+
+  '/shipping/settings':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+
+    get:
+      operationId: getCheckoutSettings
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/checkoutSettings'
+                  meta:
+                    $ref: '#/components/schemas/metaCollection'
+      description: |-
+        Get checkout settings.
+      tags:
+        - Checkout Settings
+      summary: Get Checkout Settings
```

Comment:
Remember that this is a shipping/settings endpoint. It currently just happens that the only relevant setting is applicable to checkout, but this is still shipping-settings.